### DISCUSSION TITLE
JanusGraph use case at Credit Agricole CIB

### DISCUSSION THREAD

**Discussion by balchua** [JanusGraph/janusgraph#2734] [Show and tell]
We have been working on a solution to provide our business users a way to visualize relationships between transactions.  We started to work our way using traditional relational database.  But soon realized that it will not be able to adapt to the constant changing information with different shapes and relationships.

We looked towards using Graph database and is quickly convinced that this data structure is perfect for our use case, that is to build a network of inter related transactions and its status.

We tried other graph databases, but JanusGraph won us over, due to its pluggable backend.  Since we are a heavy HBase user, JanusGraph is a perfect fit for us.  Another technology stack that we use a lot is Elasticsearch.  The JanusGraph integration with Elasticsearch is a big plus.  All these leverages existing expertise in our company.

The system we built over JanusGraph is in place for more than 2 years.  Our back office users can now quickly visualize the data they need along their complex relationships.  This gives them a productivity edge as well as a quicker response time for our customers.

**Comment by porunov** [comment]
Great use-case @balchua ! Thank you for sharing it.

**Reply by balchua** [reply]
Thanks to your great product . 😀

**Comment by jitgup** [comment]
Excellent

**Reply by balchua** [reply]
Thank you!
